build-in-container/README.md

```diff
@@ -0,0 +1,45 @@
+# Build-in-container
+The build-in-container tool provides a developer tool to quickly build Mariner packages. It is easy-to-use, and distribution and platform agnostic. It sets up a build environment in an expedient manner using a container.
+
+Please install docker on your system before using the tool.
+
+## Usage
+The run.sh script presents these options <br />
+-t                 creates container image <br />
+-b [repo_dir]      creates container, builds Mariner and outputs to out/ <br />
+-i [repo_dir]      create an interactive Mariner build container <br />
+-c [repo_dir]      cleans up the current workspace <br />
+--help             shows help on usage <br /> <br />
+
+- Unless provided, repo_dir defaults to the directory containing build-in-container tool <br />
+- Place specs to build under $repo_dir/SPECS/ <br />
+- The output from the build will be available under $repo_dir/out/ (RPMS and SRPMS) <br />
+- Logs are published under $repo_dir/logs/ <br />
+
```

Comment:
```suggestion

``` bash
# Setup the container for 1st use
cd ./CBL-MarinerTutorials
./build-in-container/run.sh -t

# Build `./SPECS/**/*.spec` automatically using the latest stable toolkit
./build-in-container/run.sh -b ./
ls ./out/RPMS/x86_64/
# hello_world_demo-1.0.0-2.cm2.x86_64.rpm  hello_world_demo-debuginfo-1.0.0-2.cm2.x86_64.rpm

# Invoke the toolkit directly
./build-in-container/run.sh -i
> #  Run the tools manually
>  make build-packages SRPM_PACK_LIST="os-subrelease"

```
```